sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/utils-service/_tests_/unit/application-event-code-during-enrolment-and-completed-utils.service.applicationEventCodeDuringEnrolmentAndCompleted.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { COEStatus } from "@sims/sims-db";
```

Comment:
Is it possible  to reduce the length of file name? `application-event-code-during-enrolment-and-completed-utils.service.applicationEventCodeDuringEnrolmentAndCompleted.spec.ts`

In windows when I switch between branches it is not letting me to delete the file

![image](https://github.com/bcgov/SIMS/assets/54600590/1d9e22a1-6591-4eed-8c21-167c4d2e194f)

Reply:
@dheepak-aot did the command `git config --system core.longpaths true` worked for you?

Reply:
Yes it worked for me. Updating the windows settings alone did not work.